package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rules-templates",
-  "version": "0.4.3",
+  "version": "0.4.4",
```

Comment:
@nicosabena was this bumped manually? (you should use this procedure: https://github.com/auth0/rules#release-notes )

Reply:
Oops